### DISCUSSION TITLE
When will version 2 be available from `go install` ?

### DISCUSSION THREAD

**Discussion by bound-variable** [ayoisaiah/f2#85] [Q&A] [answered]
I ran `go install github.com/ayoisaiah/f2/cmd/f2@latest` but it installs 1.9.1

**Comment by nervocalm** [comment]
Same here. Tried to explicitly address the module version with:
`go install github.com/ayoisaiah/f2/cmd/f2@v2.0.0`

But since is a major version change seems not to match with go.mod path:
`go: github.com/ayoisaiah/f2/cmd/f2@v2.0.0: github.com/ayoisaiah/f2@v2.0.0: invalid version: module contains a go.mod file, so module path must match major version ("github.com/ayoisaiah/f2/v2")
`

**Comment by ayoisaiah** [comment, marked as answer]
This is fixed now. Please use:

```bash
go install github.com/ayoisaiah/f2/v2/cmd/f2@latest
```
